app/backends/workflow-orchestration/work-item-updates.ts

```diff
@@ -161,11 +173,11 @@ async function handleFailedWorkItems(
   // If the response is an error then set the job status to 'failed'
   if (status === WorkItemStatus.FAILED) {
     continueProcessing = job.ignoreErrors;
-    if (!job.isComplete()) {
+    if (!job.hasTerminalStatus()) {
       let jobMessage;
 
       if (errorMessage) {
-        jobMessage = `WorkItem [${workItem.id}] failed with error: ${errorMessage}`;
```

Comment:
I removed "with error" since a lot of our error messages already contain the word error or are evidently errors.